cfme/tests/services/test_provision_stack.py

```diff
@@ -127,7 +131,16 @@ def catalog():
     catalog = Catalog(name=cat_name, description="my catalog")
     catalog.create()
     yield catalog
-    catalog.delete()
+
+
+@pytest.fixture(scope="function")
+def set_grid_view():
```

Comment:
@psav i agree we can have this added in the configure module. With this we can just pass any button group name and set the required view.